src/evergreen/task.py

```diff
@@ -18,6 +18,7 @@
 EVG_SUCCESS_STATUS = "success"
 EVG_SYSTEM_FAILURE_STATUS = "system"
 EVG_UNDISPATCHED_STATUS = "undispatched"
+EVG_TEST_STATUS = "test"
```

Comment:
Could we rename this to `EVG_TEST_STATUS_TYPE`? Since it is a status type and not a status.